akka-actor-testkit-typed/src/test/java/akka/actor/testkit/typed/javadsl/ActorTestKitTest.java

```diff
@@ -8,7 +8,7 @@
 import akka.actor.typed.javadsl.Behaviors;
 import org.junit.ClassRule;
 import org.junit.Test;
-import org.scalatestplus.junit.JUnitSuite;
+import org.scalatest.junit.JUnitSuite;
```

Comment:
This class is deprecated and we should use the one under `scalatestpulus`. However, the latter doesn't seem to be "importable". For reference: https://github.com/akka/akka-http/pull/2475 and  https://github.com/scalatest/scalatest/issues/1553

Reply:
In the `3.0.7` tag they are still using `org.scalatestplus.junit.JUnitSuite`.

Reply:
They're still referring to it from the 'deprecated' annotation, but they don't seem to be actually shipping it...